payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/MandateTextUI.kt

```diff
@@ -24,9 +24,11 @@ fun MandateTextUI(
 ) {
     Text(
         text = stringResource(element.stringResId, *element.args.toTypedArray()),
-        style = MaterialTheme.typography.body1.copy(textAlign = TextAlign.Left),
-        color = MaterialTheme.stripeColors.subtitle,
-        fontSize = 11.sp,
+        style = MaterialTheme.typography.caption.copy(
+            textAlign = TextAlign.Start,
+            fontWeight = FontWeight.Normal,
+        ),
+        color = MaterialTheme.stripeColors.placeholderText,
```

Comment:
This feels like a behavior change. I'm wondering if we should just maintain the `subtitle` coloring because changing this in a minor version potentially throws off other application's theming

Reply:
That being said we already have `Link` mandate set to `placeholderText`